### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by harryge00**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: plugin/pkg/auth/authorizer/rbac/rbac.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ import (
 
 type RequestToRuleMapper interface {
 	// RulesFor returns all known PolicyRules and any errors that happened while locating those rules.
-	// Any rule returned is still valid, since rules are deny by default.  If you can pass with the rules
+	// Any rule returned is still valid, since rules are denied by default.  If you can pass with the rules
```
Comment (deads2k) [inline plugin/pkg/auth/authorizer/rbac/rbac.go]: I think the existing "deny" is more correct.  "deny by default" as a phrase

**Comment by deads2k** [comment]
Remove the "deny" change and this lgtm.

**Comment by harryge00** [comment]
updated

**Comment by deads2k** [comment]
@k8s-bot test this

**Comment by deads2k** [comment]
@k8s-bot ok to test

**Pull request closed without merging**